examples/bevy_ggrs/Cargo.toml

```
@@ -23,10 +24,10 @@ bevy = { version = "0.9", default-features = false, features = [
   "bevy_winit",
   "render",
   "png",
-  "x11" # gh actions runners don't like wayland
+  "x11",        # gh actions runners don't like wayland
```

Can we make it not do this? Or would it work to have the comment on the line above?

---

It's happy with this 

```suggestion
  # gh actions runners don't like wayland
  "x11",
```

---

Okay, let's go with that, then :)